modules/gravatar-hovercards.php

```diff
@@ -233,12 +233,19 @@ function grofiles_extra_data() {
  */
 function grofiles_hovercards_data_html( $author ) {
 	$data = grofiles_hovercards_data( $author );
+	$hash = '';
 	if ( is_numeric( $author ) ) {
 		$user = get_userdata( $author );
-		$hash = md5( $user->user_email );
+		if ( $user ) {
+			$hash = md5( $user->user_email );
+		}
 	} else {
```

Comment:
Should this maybe be an `} elseif ( is_email( $author ) ) {` ?

https://codex.wordpress.org/Function_Reference/is_email